solution/backend/regulations/rss_feed.py

```diff
@@ -0,0 +1,85 @@
+from django.contrib.syndication.views import Feed
+from django.urls import reverse
+from datetime import datetime
+from regcore.models import Part
+from resources.models import AbstractResource
+
+
+class PartFeed(Feed):
+    def get_feed(self, obj, request):
+        feedgen = super().get_feed(obj, request)
+        feedgen.content_type = 'application/xml'  # New standard
+        return feedgen
+    title = 'Federal Register documents RSS Feed'
+    link = '/latest/feed'
+    description = 'Displays the latest federal register documents'
+
+    def processChildren(self, children, title, part, last_updated):
+        results = []
+        for child in children:
+            if child.get('type', '') == 'subpart':
+                results.append({
+                    'title': title,
+                    'part': part,
+                    'subpart': child['identifier'][0],
+                    'last_updated': last_updated,
+                })
+            # This just ends up forwarding to the subchapter page so removing for now, but we can add back if needed
+            # if child.get('type', '') == 'section':
+            #    results.append({'title': title, 'part': part, 'section': child['identifier'][1]})
+            if child['children']:
+                childResults = self.processChildren(child['children'], title, part, last_updated)
+                if childResults:
+                    results = results + childResults
+        return results
+
+    def items(self):
+        date = datetime.now()
+        title = 42
+        query = Part.objects.filter(title=title).filter(date__lte=date).order_by('name', '-date').distinct('name')
+        results = []
+        for p in query:
+            results.append({
+                'title': p.title,
+                'part': p.name,
+                'last_updated': p.last_updated,
+                'document_title': p.document['title'],
+                'document_label': 'Part {}'.format(p.document['label'][0]),
+            })
+            children = p.structure['children']
+            results = results + self.processChildren(children, p.title, p.name, p.last_updated)
+
+        return results
+
+    def item_title(self, item):
+        if 'document_label' in item:
+            return item['document_label']
+        else:
+            return '{} {} Subpart {}'.format(item['title'], item['part'], item['subpart'])
```

Comment:
both item_title and descriptin return the exact same value on the else statement.  Will this be consistent?  Would it make sense to just have a function called that takes those parameters for the return value so that way if it changes in one place it changes in both?